src/en/flamecomics/src/eu/kanade/tachiyomi/extension/en/flamecomics/FlameComics.kt

```diff
@@ -223,7 +225,7 @@ class FlameComics : HttpSource() {
                 chapter_number = chapter.chapter.toFloat()
                 date_upload = chapter.release_date * 1000
                 name = buildString {
-                    append("Chapter ${chapter.chapter.toInt()} ")
+                    append("Chapter ${chapterNumberFormat.format(chapter.chapter)} ")
```

Comment:
Wouldn't `chapter.chapter.toString().removeSuffix(".0")` be better?

Reply:
I guess it would. Didn't even think about it in this case. Thanks for sugegstion